lib/css/components/post-summary.less

```diff
@@ -309,38 +305,94 @@
 
 .s-post-summary__watched {
     background-color: var(--yellow-050);
-}
 
-.s-post-summary__deleted {
-    background-color: var(--red-050);
+    .s-post-summary--content-title > a {
+        color: var(--blue-700);
 
-    .is-deleted {
-        color: @white;
-        background-color: var(--red-500);
+        &:hover,
+        &:active,
+        &:visited {
+            color: var(--blue-700);
+        }
+    }
+
+    .s-post-summary--stats {
+        color: var(--black-500);
+    }
+
+    .s-user-card {
+        .s-user-card--link {
+            color: var(--blue-700);
+        }
+
+        .s-user-card--rep,
+        .s-user-card--time {
+            color: var(--black-600);
+        }
     }
 }
 
-.s-post-summary__ignored,
 .s-post-summary__deleted {
-    .s-post-summary--content > *:not(.s-post-summary--content-menu-button):not(.s-post-summary--meta) {
-        opacity: calc(var(--_o-disabled-static) * 1.2); // 0.5 * 1.2 = 0.6
+    background-color: var(--red-025);
+
+    .s-badge__filled {
+        color: var(--white);
+
+        .dark-mode({
+            background-color: var(--red-800)
+        });
     }
-    // TODO DEPRECATED remove `:not(.is-deleted)`
-    .s-post-summary--stats-item:not(.s-badge):not(.is-deleted) {
-        opacity: calc(var(--_o-disabled-static) * 1.2); // 0.5 * 1.2 = 0.6
-        filter: grayscale(100%);
+
+    .is-deleted {
+        color: var(--white);
+        background-color: var(--red-600);
     }
 
-    .s-post-summary--content-title a {
-        color: var(--black-600);
+    .s-post-summary--meta-tags > a {
+        background-color: var(--black-075);
+    }
```

Comment:
The interactive states of tags within deleted post summaries are still blue. Is this as expected?

Hovered tag screenshot:
![image](https://user-images.githubusercontent.com/647177/187980619-976f0c9e-0a78-427b-a5be-98faf91f70af.png)

Reply:
Thanks for catching this! It should not be blue, it should be one darker at background black-075.